lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
Do you think we should make users set the namespace on the constructor then? To make clear that the namespace is shared between all instances.

```js
var Sequelize = require('sequelize');
Sequelize.clsNamespace = 'wat';
new Sequelize();
```